Jenkinsfile

```diff
@@ -189,14 +189,16 @@ pipeline {
             node('tester') {
                 deleteDir()
                 unstash 'CIScripts'
-                // script {
-                //     try {
-                //         unstash 'testReport'
-                //         powershell script: './CIScripts/GenerateTestReport.ps1'
-                //     } finally {
-                //         stash name: 'testReport', includes: '*.xml,*.html', allowEmpty: true
-                //     }
-                // }
+                script {
+                    try {
+                        unstash 'testReport'
+                        powershell script: './CIScripts/GenerateTestReport.ps1'
```

Comment:
Can we modify it so errors in `GenerateTestReport` are not treated as no test report? Perhaps
```groovy
def hasReports
try { unstash 'testReport'; hasReports = true }
catch (Exception _) { hasReports = false }
if (hasReports) { ... }
```

Reply:
`GenerateTestReport` should not fail if there are no test reports. Script checks whether each (currently 1) XML exist.

Reply:
So I guess it could be moved to `finally` block?

Reply:
Yes. I will move it there.